dtfs-central-api/api-tests/api.js

```diff
@@ -11,7 +12,9 @@ module.exports = (app) => ({
     to: async (url) => {
       const results = [];
 
+      // eslint-disable-next-line no-restricted-syntax
```

Comment:
These two rules (Line 15 and 17) are disabled in some of the other services. Should we disable them for this service as well rather than opting out on individual lines? Or is it needed for the non-test files in `dtfs-central-api`?

Reply:
Have disabled across the micro-service and removed individual lines from across the files.